server.js

```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 const app = express();
 const cors = require("cors");
+const inventoriesRouter = require("./routes/inventoriesRouter");
 
 require("dotenv").config()
 const { PORT } = process.env;
```

Comment:
if reading port from config file failed in that case, add alternate port  using || .

Reply:
Thank you. That is a good point